conf/app_server.yaml

```diff
@@ -24,6 +24,8 @@ email_password: 19782013
 email_from: info@bh.org.il
 
 # Bucket urls
+bagnowka_bucket_url: https://s3-us-west-2.amazonaws.com/bagnowka-scraped/full
+bagnowka_bucket: full
```

Comment:
also, regarding the bagnowka_bucket_url - if you add it to configuration file, it means it has to be added on the server as well - so need to add it to deployment notes

however, in this case - if the url is always the same - maybe it's better to just hard-code it inside a constants.py file - thus simplifying the deployment